src/modules/select/index.ts

```diff
@@ -92,6 +93,8 @@ function getModule(name: string): Promise<any> | undefined {
       return import('./wallets/imtoken')
     case 'unilogin':
       return import('./wallets/unilogin')
+    case 'MYKEY':
```

Comment:
```suggestion
    case 'mykey':
```
Could you please switch this to either all lowercase or camelCase to be consistent with the rest of the wallet initialization names.